pkg/trace/stats/client_stats_aggregator.go

```diff
@@ -298,13 +301,15 @@ func (b *bucket) aggregationToPayloads() []*pb.ClientStatsPayload {
 				Stats:    groupedStats,
 			}}
 		res = append(res, &pb.ClientStatsPayload{
-			Hostname:     payloadKey.Hostname,
-			Env:          payloadKey.Env,
-			Version:      payloadKey.Version,
-			ImageTag:     payloadKey.ImageTag,
-			GitCommitSha: payloadKey.GitCommitSha,
-			ContainerID:  payloadKey.ContainerID,
-			Stats:        clientBuckets,
+			Hostname:        payloadKey.Hostname,
+			Env:             payloadKey.Env,
+			Version:         payloadKey.Version,
+			ImageTag:        payloadKey.ImageTag,
+			GitCommitSha:    payloadKey.GitCommitSha,
+			ContainerID:     payloadKey.ContainerID,
+			Stats:           clientBuckets,
+			ProcessTagsHash: payloadKey.ProcessTagsHash,
+			ProcessTags:     b.processTags[payloadKey.ProcessTagsHash],
```

Comment:
could you add one test on `client_stats_aggregator_test.go` to ensure the flushed payload contains these fields? 🙏

Reply:
it's actually covered by the test TestConvertStats in trace/agent/agent_test.go where most of client_stats_aggregator tests are, I'll check if I can find a meaningful additional test to add in `lient_stats_aggregator_test.go`